board/opentrons/ot2/rootfs-overlay/etc/nginx/nginx.conf

```diff
@@ -0,0 +1,43 @@
+user root root;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    map $http_upgrade $connection_upgrade {
+        default upgrade;
+        '' close;
+    }
+
+    server {
+        # Values in {} get replaced by environment variable during
+        # container build
+        listen [::]:31950;
+        listen 0.0.0.0:31950;
+
+        client_body_in_file_only off;
+        client_body_buffer_size  128k;
+        client_max_body_size     5M;
+
+        access_log /data/nginx-access.log;
+
+        # If nginx matches multiple location paths, it uses the one with the
+        # longest prefix, so order shouldn't matter but length of path does
+        location / {
+            proxy_http_version       1.1;
+            proxy_set_header         Upgrade $http_upgrade;
+            proxy_set_header         Connection "upgrade";
+            proxy_read_timeout       1h;
+            proxy_pass               http://unix:/tmp/aiohttp.sock;
```

Comment:
Are we planning on continuing the use of this UNIX socket?

Reply:
For now. May change later.